### ISSUE TITLE
Add badges for readme using npm-badger

### ISSUE THREAD

**Issue by arshadkazmi42** [arshadkazmi42/url-domain-name#10]
1. Install npm badger `npm install npm-badger -g`
2. Clone this repository `git clone https://github.com/arshadkazmi42/url-domain-name`
3. Go to root directory `cd url-domain-name`
4. Run npm badger `npm-badger`
5. Commit `README.md` file
6. Create Pull Request

**Comment by tarannum-2002** [comment]
I did till 3 steps 
but while 4th step to run npm-badger 
i'm getting the following error 
please help me out!

C:\Users\admin\AppData\Roaming\npm\node_modules\npm-badger\node_modules\@gh-conf\gh-conf-parse\lib\user-repo.js:22
  const currRepoData = currRepo.split('/');
                                ^

TypeError: Cannot read properties of undefined (reading 'split')
    at userRepo (C:\Users\admin\AppData\Roaming\npm\node_modules\npm-badger\node_modules\@gh-conf\gh-conf-parse\lib\user-repo.js:22:33)
    at Object.externals.generate (C:\Users\admin\AppData\Roaming\npm\node_modules\npm-badger\npm-badger.js:11:36)
    at Object.<anonymous> (C:\Users\admin\AppData\Roaming\npm\node_modules\npm-badger\index.js:5:8)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47

**Comment by arshadkazmi42** [comment]
Looks like some issue in the library. please keep this on hold. I will
check this and update you in a day or two.

Seems the npm-badger is crashing in windows
-- 


Regards,
Mohammad Arshad Kazmi
http://arshadkazmi42.github.io <http://arshad.kaspat.com>

**Comment by tarannum-2002** [comment]
Oh ok Thanks!

**Comment by arshadkazmi42** [comment]
@tarannum-2002 Can you share contents of this file `.git/config` from this directory `url-domain-name` after step 3.

Basically I need to check what is the origin URL there. it seems to be failing while parsing the origin url.

I want to see content of this section of the `.git/config` file 
![Screenshot from 2022-07-13 21-47-37](https://user-images.githubusercontent.com/4654382/178782272-3a600fbd-00c0-4a25-8ab2-c86beee3ff27.png)

**Comment by arshadkazmi42** [comment]
@tarannum-2002 I have created one issue for this here https://github.com/arshadkazmi42/npm-badger/issues/1

Can you please share the details of `.git/config` file in that issue. So we can find the root cause and get this resolved
